### DISCUSSION TITLE
detach_window new-tab with kitty @ detach-window?

### DISCUSSION THREAD

**Discussion by dyj216** [kovidgoyal/kitty#4899] [Q&A] [answered]
Is it possible to execute a version of `kitty @ detach-window` which would place the window to a new tab?

**Comment by kovidgoyal** [comment, marked as answer]
kitty @ detach-window --help

**Reply by dyj216** [reply]
Thanks, now that I carefully read it, i found it. I did not find it before, because I searched for the answer in the `--target-tab` section of the help.

For others, who did not find it:
`kitty @ detach-window -t new`
